src/app/Containers/Inbox.tsx

```diff
@@ -16,14 +16,17 @@ import { EmitterSubscription, View, ViewProps } from "react-native"
 import ScrollableTabView, { TabBarProps } from "react-native-scrollable-tab-view"
 import { createRefetchContainer, graphql, QueryRenderer, RelayRefetchProp } from "react-relay"
 
+const BIDS_TAB_INDEX = 0
+const INQUIRIES_TAB_INDEX = 1
 // Tabs
 interface TabWrapperProps extends ViewProps {
   tabLabel: string
+  isActiveTab: boolean
```

Comment:
```suggestion
  _test_isActiveTab: boolean
```
since we are only using this for the test, id prefer if we were explicit. we do have this pattern in a couple of places already, so maybe making it clear will make others avoid using it for actual code.

Reply:
I feel a little icky about this nomenclature, but I'm trusting you as an Eigen context holder, @pvinis 😅